src/pages/create-project/CreateProject.jsx

```diff
@@ -48,57 +53,137 @@ const CreateProject = (props) => {
+                                recommended={false}
+                                active={selectedDB === "mongo"} />
+                            </Col>
+                            </Row>
+                            <Row className="db-display">
+                            <Col span={3}>
+                            <StarterTemplate icon={postgresIcon} onClick={handlePostgres}
+                                heading="POSTGRESQL" desc="The world's most advanced open source database."
+                                recommended={false}
+                                active={selectedDB === "sql-postgres"} />
+                            </Col>
+                            </Row>
+                            <Row className="db-display">
+                            <Col span={3}>
+                            <StarterTemplate icon={mysqlIcon} onClick={handleMysql}
+                                heading="MYSQL" desc="The world's most popular open source database."
+                                recommended={false}
+                                active={selectedDB === "sql-mysql"} />
+                            </Col>
+                            </Row>
+                            <p style={{marginBottom:0, marginTop:0}}>Provide a connection String</p>
+                            <label style={{fontSize: 12}}>Space Cloud requires a connection string to connect to your database</label>
+                            <Form>
+                                <Form.Item >
+                                    {getFieldDecorator('connectionString', {
+                                    rules: [{ required: true, message: 'Please input a connection string' }],
+                                    initialValue: dbValue
+                                    })(
+                                    <Input placeholder="eg: mongodb://localhost:27017" />,
+                                    )}
+                                </Form.Item>
+                            </Form>
+                            <p style={{marginBottom:0, marginTop:0}}>Give your database an alias</p>
+                            <label style={{fontSize: 12}}>Alias is the name that you would use in your frontend to identify your database</label>
+                            <Form>
+                                <Form.Item>
+                                    {getFieldDecorator('alias', {
+                                    rules: [{ required: true, message: 'Please input an alias for your database' }],
+                                    initialValue: alias
+                                    })(
+                                    <Input placeholder="eg: mongo" />,
+                                    )}
+                                </Form.Item>
+                            </Form>
+                            <Link to={"/mission-control/projects/" + projectID + "/overview"} ><Button type="primary" className="db-btn">Add database</Button></Link>
```

Comment:
Don't link here directly. Call a function and do a `history.push()` inside of it. I need to upload the database config on this step. The link below will work. But this will have to be a function.